src/state/machines/turn.ts

```diff
@@ -965,7 +967,7 @@ export const TurnMachine = setup({
         },
         habitatUnlock: {
           initial: "awaitingConfirmation",
-          entry: ["stageHabitatUnlock", { type: "setAutomaticPolicyDraw", params: "habitat" }],
+          entry: [{ type: "setAutomaticPolicyDraw", params: "habitat" }, "stageHabitatUnlock"],
```

Comment:
It seems like the double state change here caused the animation duration to be 0 or something, swopping them around triggers the correct delay